metadata.json

```diff
@@ -1,39 +1,39 @@
-{
-  "name": "puppetlabs-registry",
-  "version": "2.1.0",
-  "author": "Puppet Inc",
-  "summary": "This module provides a native type and provider to manage keys and values in the Windows Registry",
-  "license": "Apache-2.0",
-  "source": "git://github.com/puppetlabs/puppetlabs-registry.git",
-  "project_page": "http://links.puppet.com/registry-module",
-  "issues_url": "https://github.com/puppetlabs/puppetlabs-registry/issues",
-  "dependencies": [
-
-  ],
-  "operatingsystem_support": [
-    {
-      "operatingsystem": "Windows",
-      "operatingsystemrelease": [
-        "Server 2008",
-        "Server 2008 R2",
-        "Server 2012",
-        "Server 2012 R2",
-        "Server 2016",
-        "Server 2019",
-        "7",
-        "8",
-        "10"
-      ]
-    }
-  ],
-  "requirements": [
-    {
-      "name": "puppet",
-      "version_requirement": ">= 4.7.0 < 7.0.0"
-    }
-  ],
-  "description": "This module provides a native type and provider to manage keys and values in the Windows Registry",
-  "pdk-version": "1.8.0",
-  "template-url": "https://github.com/puppetlabs/pdk-templates",
-  "template-ref": "heads/master-0-g7281db5"
-}
\ No newline at end of file
+{
+  "name": "puppetlabs-registry",
+  "version": "2.1.0",
+  "author": "Puppet Inc",
+  "summary": "This module provides a native type and provider to manage keys and values in the Windows Registry",
+  "license": "Apache-2.0",
+  "source": "git://github.com/puppetlabs/puppetlabs-registry.git",
+  "project_page": "http://links.puppet.com/registry-module",
+  "issues_url": "https://github.com/puppetlabs/puppetlabs-registry/issues",
+  "dependencies": [
+
+  ],
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "Windows",
+      "operatingsystemrelease": [
+        "Server 2008",
+        "Server 2008 R2",
+        "Server 2012",
+        "Server 2012 R2",
+        "Server 2016",
+        "Server 2019",
+        "7",
+        "8",
+        "10"
+      ]
+    }
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 5.5.10 < 7.0.0"
+    }
+  ],
+  "description": "This module provides a native type and provider to manage keys and values in the Windows Registry",
+  "pdk-version": "1.11.0",
+  "template-url": "https://github.com/puppetlabs/pdk-templates",
+  "template-ref": "1.11.0-0-g67637d4"
```

Comment:
You need to run `pdk convert --template-url https://github.com/puppetlabs/pdk-templates --template-ref master`.

Due to changes introduced in pdk 1.10.0, using pdk update on a module using the old template-url syntax (a plain url) will default the target ref to whatever the latest release for the version of pdk you are using is. To fix this you need to run pdk convert --template-url https://github.com/puppetlabs/pdk-templates --template-ref master. This will add the #master ref to the url and using pdk update after this will work as expected. This is just a one off change basically.